src/lazy/text/buffer.rs

```diff
@@ -1021,6 +1042,37 @@ impl<'data> TextBufferView<'data> {
         Ok((remaining, matched))
     }
 
+    pub fn match_struct_1_1(self) -> IonMatchResult<'data> {
```

Comment:
🗺️ This is basically a copy/paste of `match_struct`, but it uses `RawTextStructIterator_1_1`.